client/js/dashboard.js

```diff
@@ -108,6 +108,8 @@ if (Element && !Element.prototype.matches) {
             console.error(err);
             ErrorIndicator.show();
         });
+    }).catch(function(e) {
+        ErrorIndicator.show();
```

Comment:
What about the console.error happening down in kernel.js? Was thinking it's kinda weird that we do some logging in one place and other logging/error handling in another? Should that move up too?